ui/jobs/jobs.js

```diff
@@ -2786,6 +2803,19 @@ class Bars {
 
     for (const func of this.changeZoneFuncs)
       func(e);
+
+    this.isPVPZone = 0;
+    if (zoneInfo) {
+      // 6 => pvp content, 250 => Wolves' Den Pier
+      if (zoneInfo.contentType == 6 || e.zoneID == 250)
```

Comment:
`if (zoneInfo.contentType === ContentType.Pvp || e.zoneID === ZoneId.WolvesDenPier)`

Reply:
`Uncaught ReferenceError: ZoneId is not defined`, Maybe we need import it somewhere?